.github/runners/README.md

```diff
@@ -2,41 +2,73 @@
 
 How to provision GitHub Actions self-hosted runners for Flux conformance testing.
 
-## ARM64 Instance specs
+## Instance specs
 
 In order to add a new runner to the GitHub Actions pool,
-first create an instance on Oracle Cloud with the following configuration:
-- OS: Canonical Ubuntu 20.04
-- Shape: VM.Standard.A1.Flex
-- OCPU Count: 2 
-- Memory (GB): 12
-- Network Bandwidth (Gbps): 2
-- Local Disk: Block Storage Only  
+first create a server on Equinix with the following configuration:
```

Comment:
I am unclear on whether this describes how to restore the runners, given the Equinix instances, or how to make it all from scratch. This says "first create a server ...", but the table below mentions two instances. So is the table the current state, and the instruction above about how to recreate the current state? I suggest moving the table of desired (or current) state to the top, with the instructions for creating it beneath (and carefully titled).